### ISSUE TITLE
Order seems to matter in insert into

### ISSUE THREAD

**Issue by silasdavis** [adelsz/pgtyped#79]
If inserting a row with a subset of the columns if the order of my subset does not match the declared order of the table columns.

For example if I insert a row with just four columns out of many set in the same order as they were declared in the table the query works as expected:

```typescript
const insertParamUpdate = sql<IInsertParamUpdateQuery>`
    insert into data.event_log
    (_chainid, _height, _txindex, storage_address  )
    values
    $$params(_chainid, _height, _txindex,  storage_address);
`;
```

But if I swap `storage_address` in the column list nad the values object:

```typescript
const insertParamUpdate = sql<IInsertParamUpdateQuery>`
    insert into data.event_log
    (_chainid, storage_address, _height, _txindex )
    values
    $$params(_chainid, storage_address _height, _txindex);
`;
```

This fails with a `invalid input for type bigint`. This makes it seem like `storage_address` is being provided for a `_height` value.

It's late now but a test case shouldn't be too hard if this is a general bug.

**Comment by adelsz** [comment]
Hm, this is interesting.
I can also see a comma missing on line 5 in the second code block, is that a typo?

**Comment by silasdavis** [comment]
Ah yes it is, I've provided a test case in that pr

**Comment by adelsz** [comment]
Thanks for creating a test case! Fixed in v0.7.2.
